pkg/shared/families/infofinder/sshtopology/scanner.go

```diff
@@ -77,11 +77,7 @@ func (s *Scanner) Run(sourceType utils.SourceType, userInput string) error {
 		defer cleanup()
 
 		var errs []error
-		homeUserDirs, err := getHomeUserDirs(fsPath)
-		if err != nil {
-			// Collect the error and continue.
-			errs = append(errs, fmt.Errorf("failed to get home user dirs: %w", err))
-		}
+		homeUserDirs := getHomeUserDirs(fsPath)
 		s.logger.Debugf("Found home user dirs %+v", homeUserDirs)
```

Comment:
nit: this can be moved to line 113 just to be closer to the place where it's actually being used